packages/mdc-fab/README.md

```diff
@@ -87,6 +88,7 @@ const fabRipple = new MDCRipple(document.querySelector('.mdc-fab'));
 <button class="mdc-fab mdc-fab--extended">
   <span class="material-icons mdc-fab__icon">add</span>
   <span class="mdc-fab__label">Create</span>
+  <div class="mdc-fab__ripple"></div>
 </button>
```

Comment:
Update README to move ripple to first child.

Reply:
Done, thanks for catching that!